.github/workflows/release.yml

```diff
@@ -100,7 +100,7 @@ jobs:
           vaultRoleId: ${{ secrets.VAULT_ROLE_ID }}
           vaultSecretId: ${{ secrets.VAULT_SECRET_ID }}
           pipeline: apm-agent-java-release
-          pipelineVersion: ${{ env.TAG_NAME }}
+          pipelineCommit: ${{ env.TAG_NAME }}
```

Comment:
This caused a regression. Though https://github.com/elastic/apm-pipeline-library/pull/2450 will fix it and make the default behaviour with a higher precedence.